paypal-commercetools-custom-application/src/constants.ts

```diff
@@ -1,8 +1,10 @@
 // Make sure to import the helper functions from the `ssr` entry point.
 import { entryPointUriPathToPermissionKeys } from '@commercetools-frontend/application-shell/ssr';
 
-export const entryPointUriPath = '${env:ENTRY_POINT_URI_PATH}';
```

Comment:
only out of sheer interest: why did you change this piece of code? Did you experience problems with permissions otherwise?

Reply:
Mainly I have followed the guide https://docs.commercetools.com/custom-applications/deployment-examples/commercetools-connect but I think it is this way in the guides because of possible permissions issue.